### PR TITLE
GenerateSequence function

### PR DESCRIPTION
**Pull request by kondorski**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: generate-sequence
🏅 Granted 5 XP for @kondorski

**Comment by ziobron** [comment]
⏱ 3 XP extra for @kondorski for on-time delivery

File: homework/generate-sequence/generateSequence.hpp [hunk 1]
```diff
@@ -2,6 +2,17 @@
...
+    int temp {0};
+    if(count <= 0){
+        return numbers;
+    }
+    else{
+        do{
+            temp += step;
+            numbers.push_back(temp);
+            count--;
+        }while(count > 0);
```
Comment (nauka-programowania-MA) [inline homework/generate-sequence/generateSequence.hpp:15]: Add space after `if`, `for`, `while` `function` etc...

File: homework/generate-sequence/generateSequence.hpp [hunk 2]
```diff
@@ -2,6 +2,17 @@
...
 std::vector<int> generateSequence(int count, int step) {
-    // TODO: Implement me :)
-    return {};
+    std::vector<int> numbers{};
+    int temp {0};
+    if(count <= 0){
+        return numbers;
+    }
+    else{
+        do{
```
Comment (nauka-programowania-MA) [inline homework/generate-sequence/generateSequence.hpp:11]: Add space after `if`, `for`, `while` `function` etc...

File: homework/generate-sequence/generateSequence.hpp [hunk 3]
```diff
@@ -2,6 +2,17 @@
 #include <vector>
 
 std::vector<int> generateSequence(int count, int step) {
-    // TODO: Implement me :)
-    return {};
+    std::vector<int> numbers{};
```
Comment (nauka-programowania-MA) [inline homework/generate-sequence/generateSequence.hpp:5]: Reserve memory when you know how many elements will be inside vector.

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**